frontend/pages/shopping-lists/_id.vue

```diff
@@ -188,6 +188,7 @@
 import draggable from "vuedraggable";
 
 import { defineComponent, useAsync, useRoute, computed, ref, watch } from "@nuxtjs/composition-api";
+import { onUnmounted } from "vue-demi";
```

Comment:
This ins't the correct import 'vue-demi' is only in development. I think all of this needs to come from the Nuxt composition API. I thought I had a linter rule for this, but maybe I missed this package.

Reply:
Ah okay, I wondered what that was but I saw it somewhere else. Will fix, thanks!

Reply:
You're absolutely right, it's all over the place. No idea how any of that is working in production... Will look at that in a PR shortly.